From 8bd4295d0356b95a24802fbc81dd459711750125 Mon Sep 17 00:00:00 2001
From: Mark Hatle <mark.hatle@amd.com>
Date: Fri, 18 Sep 2026 22:02:24 -0600
Subject: [PATCH] CMakeLists.txt: Add cmake_minimum_required

Fix for configure failure:

| CMake Error in CMakeLists.txt:
|   No cmake_minimum_required command is present.  A line of code such as
|
|     cmake_minimum_required(VERSION 4.3)
|
|   should be added at the top of the file.  The version specified may be lower
|   if you wish to support older CMake versions for this project.  For more
|   information run "cmake --help-policy CMP0000".

Upstream-Status: Pending

Signed-off-by: Mark Hatle <mark.hatle@amd.com>
---
 fal/src/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fal/src/CMakeLists.txt b/fal/src/CMakeLists.txt
index 86d27648..71b0c1aa 100644
--- a/fal/src/CMakeLists.txt
+++ b/fal/src/CMakeLists.txt
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: MIT
 #
 ###############################################################################
+cmake_minimum_required (VERSION 3.5)
 
 if(NOT ${YOCTO})
 	add_library(aiefal INTERFACE)
-- 
2.34.1

